raocp/core/cones.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+
+
+class Uni:
+    """
+    The universe cone (R^n)
+    """
+
+    def __init__(self):
```

Comment:
Why not allow the user to specify a dimension as an optional argument? Also, shouldn't it default to `None`?